src/odemis/gui/comp/stream.py

```diff
@@ -2154,11 +2154,48 @@ def __init__(self, parent,
         """
```

Comment:
The docstring for `FastEMOverviewSelectionPanel` is incorrect. It states it'd for the calibration. Could you update it?